### PR TITLE
Form resources

### PR DESCRIPTION
**Pull request by cfarm**
Here are the changes Dan made for KYBO landing pages

### CODE REVIEW

**Comment by cfarm** [comment]
@danny8000 once you get the final templates into this branch, let me know so I can handle their release by manually adding the code to the `form-explainers-release` branch

**Comment by mthibos** [comment]
So, it is in our repo?

**Comment by danny8000** [comment]
Hi @cfarm,

The form resources branch is ready to be merged into master.

Thanks for your help,

Dan

**Comment by cfarm** [comment]
:+1: Lookin goooooood

File: src/static/css/module/form-resources.less [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+            }
+        });
+    }
+
+
+    .hero__mortage-closing {
+
+        .respond-to-max(660px, {
+            .hero_img {
+                background-image:url('/static/img/form-resources_mortgage-closing-hero-small.png');
```
Comment (cfarm) [inline src/static/css/module/form-resources.less]: these URLs will need to be fixed to follow this file path so they work across environments:

`'../img/form-resources-example-image.png'`

(currently they don't work on the dev server)

**Pull request merged**